src/node/history.h

```diff
@@ -369,6 +382,14 @@ namespace ccf
       nodes(nodes_)
     {}
 
+    bool update_r_tree()
+    {
+      auto consensus = store.get_consensus();
+      if (!consensus || consensus->type() == ConsensusType::Pbft)
+        return true;
```

Comment:
Could you add `{}` around this line?

Reply:
around the `return true`? sure but we don't generally use curly braces for single line if statements

Reply:
I thought we agreed to use curly braces for new code added?

Reply:
I think that was for if/else's but not 100% sure